react/components/Charts/LineChart/constants.ts

```diff
@@ -0,0 +1,15 @@
+import { LineProps } from 'recharts'
+const colors = [
+    '#EE7850',
+    '#4CAABF',
+    '#9F50B6',
+    '#479A5E'
+]
+type DefaultLineProps = Pick<LineProps, 'type' | 'strokeWidth' | 'dot'>
+const defaultProps: DefaultLineProps = {
+    type: 'monotone',
+    strokeWidth: 3,
+    dot: false,
+}
+
+export { colors, defaultProps }
```

Comment:
Make your exports explicit on the constant declaration looks better than a single export at the file's end. (IMO)

Reply:
I agree with Matheus...
You can do like `export const defaultProps: DefaultLineProps = {
    type: 'monotone',
    strokeWidth: 3,
    dot: false,
}`